cfme/networks/floating_ips.py

```diff
@@ -0,0 +1,67 @@
+import attr
+from navmazing import NavigateToAttribute
+
+from cfme.common import WidgetasticTaggable
+from cfme.exceptions import ItemNotFound
+from cfme.modeling.base import BaseCollection, BaseEntity, parent_of_type
+from cfme.networks.views import FloatingIpDetailsView, FloatingIpView
+from cfme.utils.appliance.implementations.ui import navigator, CFMENavigateStep, navigate_to
+
+
+@attr.s
+class FloatingIp(WidgetasticTaggable, BaseEntity):
+    """Class representing flating ips"""
```

Comment:
Spelling.   You meant floating ips, I'm sure.   Probably should be "floating IP's".

Reply:
These are special IP's that get larger over time :P